docs/app/Examples/views/Statistic/Types/StatisticGroupExample.js

```diff
@@ -0,0 +1,24 @@
+import React, {Component} from 'react';
+import {Statistic} from 'stardust';
+const {Statistics, Label, Value} = Statistic;
```

Comment:
The issue there was, the label can be on top or on bottom.  So we had to either make a `topLabel` and `bottomLabel` prop (and probably a `label` prop that defaulted to top).  Or, go our sub-component approach.  Knowing this, is your preference the same?